ansible/provision-and-execute-tests/playbook.yml

```diff
@@ -0,0 +1,58 @@
+
+- name: Provision test executor instances and execute tests
+  hosts: linux
+  vars:
+    node_version: 16.3.0
+
+    # The following information is populated using the gathered inventory variables
+    fluent_bit_package_name: "{{ tags.fb_package_name }}"
+    pr_number: "{{ tags.pr_number }}"
+  environment:
+    NEW_RELIC_API_KEY: "{{ lookup('ansible.builtin.env', 'NEW_RELIC_API_KEY') }}"
```

Comment:
Is this necessary if they already exist in the environment (`ansible.builtin.env`) and thus accessible from whitin tasks (like `newrelic.newrelic_install`)? I guess it would make sense to set them as `vars` instead so that they can be injected to tasks via vars?

Reply:
The [newrelic install Ansible module](https://github.com/newrelic/ansible-install) actually [expects](https://github.com/newrelic/ansible-install#environment-variables) to find these three environment variables in the _managed host_ environment. Note this is not the Ansible controller environment! What we actually do here is to fill up the _managed host_ environment by reading the values from the _Ansible controller_ environment using the _lookup_ plugin.